src/webpub_manifest_parser/errors.py

```diff
@@ -36,7 +33,7 @@ def error_message(self):
         :return: Description of the error
         :rtype: str
         """
-        return six.text_type(str(self))
+        return str(str(self))
```

Comment:
Good catch... since `six.text_type` is just an alias for `str` in Python 3, I think this is a accurate translation of what the code was doing before. No need for it though. Will make PR to remove.